server/config/constants.go

```diff
@@ -29,6 +33,10 @@ const (
 	WebSocketEventPluginDisabled  = "plugin_disabled"
 )
 
+const (
+	RequestTimeout = time.Second * 30
```

Comment:
2/5 need a more-intention revealing name - what request?

Reply:
Do you have a suggestion?